rtbkit/core/banker/go_account.cc

```diff
@@ -208,6 +208,15 @@ GoAccounts::updateBalance(const AccountKey &key, int64_t newBalance)
     account->router->balance = newBalance;
 }
 
+int64_t
```

Comment:
for now they are saved as just int in redis. but that's just a function i'm using to track balance with redis.